actions/pr-slack-alert/action.yml

```diff
@@ -0,0 +1,120 @@
+name: pr-slack-alert
+description:
+  "Send Slack alert to a specific user group in a channel from Github PRs"
+
+inputs:
+  # general inputs
+  checkout-repo:
+    description: "enable git checkout repo"
+    required: false
+    default: "true"
+  checkout-repo-fetch-depth:
+    description: "number of commits to fetch"
+    required: false
+    default: "0"
+  # slack inputs
+  slack-channel-id:
+    description: "slack channel id to send the alert to"
+    required: true
+  slack-user-group-id:
+    description: "slack user group id to ping with the alert"
+    required: true
+  slack-bot-token:
+    description:
+      "slack bot api token - the token requires `chat:write` permission"
+    required: true
+  # github inputs
+  ignore-pr-creators:
+    description: "comma-separated list of github usernames to skip alerts for"
+    required: false
+    default: "github-actions[bot],dependabot[bot],renovate[bot]"
+  match-reviewer-team:
+    description:
+      "will only send alert if the team name is included in the reviewers list"
+    required: false
+    default: ""
+  pr-event-type:
+    description:
+      "filter to trigger on specific PR event type - if set to empty it will
+      trigger on every type"
+    required: false
+    default: ""
```

Comment:
Can we include some example types here and maybe some a github docs link for event types in the README?